exporter/exporterhelper/tracehelper.go

```diff
@@ -201,41 +168,19 @@ func NewOTLPTraceExporter(
 	}, nil
 }
 
-// withMetrics wraps the current pusher into a function that records the metrics of the
-// pusher execution.
-func (p otlpTraceDataPusher) withMetrics() otlpTraceDataPusher {
+// withObservability wraps the current pusher into a function that records
+// the observability signals during the pusher execution.
+func (p otlpTraceDataPusher) withObservability(exporterName string) otlpTraceDataPusher {
```

Comment:
Note to myself: I also touched this line (working on OTLP), need to resolve a conflict here.